composer.json

```diff
@@ -30,6 +30,7 @@
     }
   ],
   "version": "dev-develop",
+  "version-release": "#release_version#",
```

Comment:
I belive, we should simply replace "dev-develop" directly and drop `*-release` keys completely, it will simplify the configs and mistakes in all repos

Reply:
If replacing atk4 dev-develop is the only thing to replace, then maybe, but idea behind using `*-release` was that it can be used for any other possible changes which we need when releasing too.

Reply:
I am quite sure yes, it is the only thing to replace. We should do that, to simplify that configs and prevent mistakes.

For development, there is `require-dev`, which should be enought :)

Reply:
Agree with Michael and better use jq for editing